### ISSUE TITLE
[Help] First message upon connection is in Chinese - how to change?

### ISSUE THREAD

**Issue by shulist** [emqx/MQTTX#687]
#### Describe the problem
When connecting to an MQTT Broker I get a confirmation message in Chinese. The device is configured for English as is MQQTX. Can someone advise on how to get this message in English?
#### More detail (optional)
Linux Mint 20.1, MQTTX 1.6.0
####Additional context or screenshots.
![MQTTX_Chinese](https://user-images.githubusercontent.com/6595819/133364814-3a5caaf5-887e-4137-b3cf-8d95d457b02d.png)

**Comment by ysfscream** [comment]
@shulist Hi, this has nothing to do with the language set by MQTT X, this is a message sent by someone else.

**Comment by shulist** [comment]
@ysfscream I don't understand your answer. This is a new install of MQTTX on a Linux machine that also hosts the MQTT broker. There are no other users and I am accessing the MQTT broker from the same machine it is hosted on. Essentially this is a closed loop. How could "someone else" send this message. And it only occurs when I make a connection to the broker.

**Comment by shulist** [comment]
Hmmm... @ysfscream just thought about this problem and was wondering if the message is generated by the broker itself. In thinking about the logic it seems more likely this is the case. I'll go off to Eclipse/Mosquitto and investigate there. I'll report back later.

**Comment by shulist** [comment]
@ysfscream: It seems that somewhere in the great mass of code that makes up either Mosquitto or MQTTX someone had entered the  _**Chinese message**_ with a `retain flag on` and thus it was propagated each time a connection occurred. This had to be prior to the application being released. My vote was that Mosquitto was the culprit.

**Comment by ysfscream** [comment]
Yes, I would say the same thing, the content here is from a message posted by the MQTT Broker, so the person who published this retain message is a Chinese. LOL :)
